src/Domain/Value/Story.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Storyblok-Api.
+ *
+ * (c) SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SensioLabs\Storyblok\Api\Domain\Value;
+
+use OskarStark\Value\TrimmedNonEmptyString;
+use Safe\DateTimeImmutable;
+use Webmozart\Assert\Assert;
+
+final readonly class Story
```

Comment:
Like it with one thing because this is now without any of our plugins we use like the translatable slugs. how do we handle this? Because we need the translatable slugs etc.

Reply:
Will add it as it's part of the official documentation 
(i let it in the comments for now, because this would require some more code)

The other plugins only impact "content" i think